tests/leak-tests/pom.xml

```diff
@@ -0,0 +1,488 @@
+            <groupId>com.github.d-leak.d-leak</groupId>
+            <artifactId>maven-plugin</artifactId>
+            <version>0.2</version>
+            <executions>
+               <execution>
+                  <phase>generate-sources</phase>
+                  <id>find-native</id>
+                  <goals>
+                     <goal>install</goal>
+                  </goals>
+                  <configuration>
+                     <target>${project.basedir}/target/lib</target>
+                     <lib>libdleak.dll</lib>
+                  </configuration>
+               </execution>
+            </executions>
+         </plugin>
+         <plugin>
+            <artifactId>maven-resources-plugin</artifactId>
+            <configuration>
+               <nonFilteredFileExtensions>
+                  <nonFilteredFileExtension>jks</nonFilteredFileExtension>
+               </nonFilteredFileExtensions>
+            </configuration>
+            <executions>
+               <execution>
+                  <id>copy-security-resources</id>
+                  <phase>validate</phase>
+                  <goals>
+                     <goal>copy-resources</goal>
+                  </goals>
+                  <configuration>
+                     <outputDirectory>${basedir}/target/test-classes</outputDirectory>
+                     <resources>
+                        <resource>
+                           <directory>../security-resources</directory>
+                        </resource>
+                     </resources>
+                  </configuration>
+               </execution>
+            </executions>
+         </plugin>
+         <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-jar-plugin</artifactId>
+            <executions>
+               <execution>
+                  <phase>test</phase>
+                  <goals>
+                     <goal>test-jar</goal>
+                  </goals>
+               </execution>
+            </executions>
+         </plugin>
+         <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+               <skipTests>false</skipTests>
+               <argLine>-agentpath:${project.basedir}/target/lib/libdleak.dll -Djgroups.bind_addr=::1 ${activemq-surefire-argline} ${its-surefire-extra-args} -Dorg.apache.activemq.SERIALIZABLE_PACKAGES="java.lang,javax.security,java.util,org.apache.activemq,org.fusesource.hawtbuf"</argLine>
+            </configuration>
+         </plugin>
+      </plugins>
+   </build>
+   <repositories>
+      <repository>
+         <id>jitpack.io</id>
+         <url>https://jitpack.io</url>
+      </repository>
+   </repositories>
+   <pluginRepositories>
+      <pluginRepository>
+         <id>jitpack.io</id>
+         <url>https://jitpack.io</url>
+      </pluginRepository>
+   </pluginRepositories>
```

Comment:
Would much prefer to see this deployed to central than relying on an 'automagically built code' third party repo.

Reply:
agreed.. I will try to do it.

Reply:
this is done...